test_unstructured_ingest/test-ingest-parameters.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+cd "$SCRIPT_DIR"/.. || exit 1
+
+PYTHONPATH=. ./unstructured/ingest/main.py \
+    --metadata-exclude filename,file_directory,metadata.data_source.date_processed \
+    --local-input-path example-docs/english-and-korean.png \
+    --structured-output-dir parameterized-ingest-output \
+    --partition-strategy hi_res \
+    --ocr-languages eng+kor \
```

Comment:
Perhaps rename this file to `test-ingest-local-single-file.sh` , since it is also testing the local connector with a single file?
It would also be good to store the json output in test_unstructured_ingest/expected-structured-output/ for validation, similar to https://github.com/Unstructured-IO/unstructured/blob/main/test_unstructured_ingest/test-ingest-biomed-path.sh